### PR TITLE
lightboxes-openStandard

### PR DESCRIPTION
**Pull request by Tommyhudon**
Added information in lightboxes for Open Standards

### CODE REVIEW

**Comment by Tommyhudon** [comment]
Added "Reference URL" and "Started using"

File: _data/schemaStandard.yaml [hunk 1]
```diff
@@ -37,7 +37,9 @@ administrations:
         name:
           en: str()
           fr: str()
-    status: enum('deprecated', 'mandatory', 'preferred')
+    status:
+      en: enum('deprecated', 'mandatory', 'preferred')
+      fr: enum('obsolete','mandatoire','favoris')
```
Comment (smellems) [inline _data/schemaStandard.yaml]: Translations are here https://github.com/canada-ca/ore-ero/blob/ae2eacf96f8bb893bfa0b7cff058d408487efc92/_data/i18n/form.yml#L334-L342

File: _data/standard/html5.yml [hunk 2]
```diff
@@ -32,10 +32,12 @@ administrations:
       created: '2018-01-23'
       metadataLastUpdated: '2018-11-15'
     references:
-      - URL:
```
Comment (smellems) [inline _data/standard/html5.yml]: There can be more than one reference.  See schema https://canada-ca.github.io/ore-ero/en/open-standard-schema.html

File: _data/schemaStandard.yaml (hunk 1 above)
Comment (Tommyhudon) [reply _data/schemaStandard.yaml]: So I change mandatory and preferred for maintained and retired ?

**Pull request closed without merging**